### ISSUE TITLE
Re11-worldwise > src > components > Form | navigate(-1) not working as expected

### ISSUE THREAD

**Issue by devYuraKim** [devYuraKim/react#18]
- **code in question**: `<Button onClick={() => navigate(-1, { replace: true })} type="back">`
- **expected action**: go back to the previous URL
- **actual action**: `Form` component being re-rendered

```
function Form() {

  const navigate = useNavigate();

  return (
    <form className={styles.form}>
      <div className={styles.row}>
        <label htmlFor="cityName">City name</label>
        <input
          id="cityName"
          onChange={(e) => setCityName(formatDate(e.target.value))}
          value={cityName}
        />
        {/* <span className={styles.flag}>{emoji}</span> */}
      </div>

      <div className={styles.row}>
        <label htmlFor="date">When did you go to {cityName}?</label>
        <input
          id="date"
          onChange={(e) => setDate(e.target.value)}
          value={date}
        />
      </div>

      <div className={styles.row}>
        <label htmlFor="notes">Notes about your trip to {cityName}</label>
        <textarea
          id="notes"
          onChange={(e) => setNotes(e.target.value)}
          value={notes}
        />
      </div>

      <div className={styles.buttons}>
        <Button type="primary">Add</Button>
        <Button onClick={() => navigate(-1, { replace: true })} type="back">
          &larr; Back
        </Button>
      </div>
    </form>
  );
}
```

**Comment by devYuraKim** [comment]
TL;DR: default `form` and `button` interaction
---
- **Point1**: Default `button` behavior in `form`
Inside a form, a `button` with `type="submit"` (or no type specified, as the default is submit) will trigger the form submission when clicked. This submission might cause a page refresh or re-render of the form component

- **Point2**: `e.preventDefault()`
It prevents the default action associated with the event. In this case, it prevents the default form submission action triggered by the button click.

```
<Button
   onClick={(e) => {
            e.preventDefault();
            navigate(-1);
          }}
   type="back"
>
```
